frontend/src/metabase/components/SelectList/BaseSelectListItem.jsx

```diff
@@ -13,7 +13,7 @@ const propTypes = {
   size: PropTypes.oneOf(["small", "medium"]),
   className: PropTypes.string,
   children: PropTypes.node.isRequired,
-  as: PropTypes.element,
+  as: PropTypes.any,
```

Comment:
Not sure what the right prop type should go here, but the error was annoying enough for me to do this 🙈